SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialLatencyTracer.cpp

```diff
@@ -123,7 +123,7 @@ bool USpatialLatencyTracer::IsValidKey(const TraceKey Key)
 	return TraceMap.Find(Key);
 }
 
-TraceKey USpatialLatencyTracer::GetTraceKey(const UObject* Obj, const UFunction* Function)
+TraceKey USpatialLatencyTracer::GetAndUntrackTrace(const UObject* Obj, const UFunction* Function)
```

Comment:
I like the sentiment of the function name change, but think it's a little confusing, as we're still tracking the trace. What about `RetrievePendingTrace` ?